docs/docs/cloud/how-tos/remote_graph.md

```diff
@@ -0,0 +1,150 @@
+# How to interact with the deployment using RemoteGraph
+
+`RemoteGraph` is an interface that allows you to interact with your LangGraph Platform deployment as if it were a regular `CompiledGraph` instance. This guide shows you how you can initialize a `RemoteGraph` and interact with it.
+
+## Setup
+
+When initializing a `RemoteGraph`, you must always specify:
+
+- `name`: the name of the graph you want to interact with. This is the same graph name you use in `langgraph.json` configuration file for your deployment. 
+- `api_key`: a valid LangSmith API key. Can be set as an environment variable (`LANGSMITH_API_KEY`) or passed directly via the `api_key` argument. The API key could also be provided via the `client` / `sync_client` arguments, if `LangGraphClient` / `SyncLangGraphClient` were initialized with `api_key` argument.
+
+Additionally, you have to provide one of the following:
+
+- `url`: URL of the deployment you want to interact with. If you pass `url` argument, both sync and async clients will be created using the provided URL, headers (if provided) and default configuration values (e.g. timeout, etc).
+- `client`: a `LangGraphClient` instance for interacting with the deployment asynchronously (e.g. using `.astream()`, `.ainvoke()`, `.aget_state()`, `.aupdate_state()`, etc.)
+- `sync_client`: a `SyncLangGraphClient` instance for interacting with the deployment synchronously (e.g. using `.stream()`, `.invoke()`, `.get_state()`, `.update_state()`, etc.)
+
+!!! Note
+
+    If you pass both `client` or `sync_client` as well as `url` argument, they will take precedence over the `url` argument. If none of the `client` / `sync_client` / `url` arguments are provided, `RemoteGraph` will raise a `ValueError` at runtime.
+
+
+### Using URL
+
+=== "Python"
+
+    ```python
+    from langgraph.pregel.remote import RemoteGraph
```

Comment:
Should this be exposed under a non pregel namespace?

```python
from langgraph.pregel.remote import RemoteGraph
```

Reply:
had the same question, let me ask a broader group